### PR TITLE
Add lossy scheduler for queue 7

### PR DESCRIPTION
**Pull request by bingwang-ms**
#### Why I did it
This PR is to add scheduler for Queue 7 for all ports if `tunnel_qos_remap_enable` is `True`.
PR https://github.com/sonic-net/sonic-buildimage/pull/10176 update the default `DSCP_TO_TC_MAP` to map DSCP 48 to TC 7, and TC 7 is mapped to Queue 7. However, the scheduler for Queue  7 is missing.
This PR fixed the issue by adding lossy scheduler for Queue 7.
Thanks @stephenxs  reporting this issue.

#### How I did it
This PR fixed the issue by adding lossy scheduler for Queue 7.

#### How to verify it
Verified by UT.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

#### Description for the changelog

This PR is to add scheduler for Queue 7 for all ports if `tunnel_qos_remap_enable` is `True`.

### CODE REVIEW

File: src/sonic-config-engine/tests/sample_output/py3/qos-arista7260-dualtor.json [hunk 1]
```diff
@@ -1849,202 +1849,394 @@
...
         },
+        "Ethernet252|7": {
+            "scheduler": "scheduler.0"
+        },
         "Ethernet0|6": {
             "scheduler": "scheduler.0"
         },
         "Ethernet4|6": {
             "scheduler": "scheduler.0"
-        },
```
Comment (stephenxs) [inline src/sonic-config-engine/tests/sample_output/py3/qos-arista7260-dualtor.json:2047]: fix indent?

File: files/build_templates/qos_config.j2 [hunk 2]
```diff
@@ -233,6 +233,7 @@
         "global": {
             "dscp_to_tc_map"  : "AZURE"
         }{% if PORT_ACTIVE %},{% endif %}
+
```
Comment (stephenxs) [inline files/build_templates/qos_config.j2:236]: do we need it?

**Review by stephenxs — COMMENTED** [review]
Minor comments.

File: files/build_templates/qos_config.j2 (hunk 2 above)
Comment (bingwang-ms) [reply files/build_templates/qos_config.j2:236]: Yes, I found the generated json file will be like below without the empty line.
```
"global": {
            "dscp_to_tc_map"  : "AZURE"
        }, "Ethernet0": {
            "dscp_to_tc_map": "[DSCP_TO_TC_MAP|AZURE]",
            "pfc_enable": "3,4",
            "pfc_to_queue_map": "[MAP_PFC_PRIORITY_TO_QUEUE|AZURE]",
            "pfcwd_sw_enable": "3,4",
            "tc_to_pg_map": "[TC_TO_PRIORITY_GROUP_MAP|AZURE]",
            "tc_to_queue_map": "[TC_TO_QUEUE_MAP|AZURE]"
        },
```
So I added this empty line. 
There is also empty line after other `{% if not loop.last %},{% endif %}`

File: src/sonic-config-engine/tests/sample_output/py3/qos-arista7260-dualtor.json (hunk 1 above)
Comment (bingwang-ms) [reply src/sonic-config-engine/tests/sample_output/py3/qos-arista7260-dualtor.json]: Fixed. The indent change is caused by manually editing. The template is not changed.

**Review by stephenxs — APPROVED** [review]

**Review by neethajohn — APPROVED** [review]

**Pull request merged**